torch/_inductor/fx_passes/post_grad.py

```diff
@@ -654,6 +655,8 @@ def reinplace_scatters(graph):
             mutated_inputs.add(node.args[0])
 
     def can_inplace(node, mutated_arg):
+        if isinstance(mutated_arg, (list, tuple)):
+            return all(can_inplace(node, arg) for arg in mutated_arg)
```

Comment:
This is incorrect. Notice how can_inplace could `graph.erase_node(copy_node)`.

Reply:
well, I do think it's not great that `can_inplace` does that :P

Reply:
yeah, when inplace is not recursive, the solution is returning a ternary value. With this recursive version you would need ternary value with a list

Reply:
Good catch. The cases I'm dealing with never have a `copy_` epilogue so I overlooked that.

Reply:
@oulgen I think a cleaner way is to improve the overall handling of ops that mutates multiple inputs. However, that may cause too much conflict with your in-flight changes. I opted to simply move the copy epilogue removal out of `can_inplace` for now. Let me know what you think.